src/drivers/amqp_interop/Queue.php

```diff
@@ -256,9 +257,11 @@ public function listen()
             }
 
             return true;
-        });
+        };
 
-        $this->context->consume();
+        $subscriptionConsumer = $this->context->createSubscriptionConsumer();
```

Comment:
Are these available in `^0.8`?

Reply:
I was about to ask about it. No there is no such method in 0.8.
Do you have an idea of how tests for PHP 5.6 and 7.0 with the new code and with amqp-lib 0.8 pass now?  Am I missing something?

Reply:
Means this code isn't called in tests.

Reply:
It should.

Reply:
amqp interop tests are only skipped for PHP 5.5

Reply:
Then no idea :(

Reply:
Actually, this method is available in 0.8
https://github.com/php-enqueue/amqp-lib/blob/0.8.42/AmqpContext.php#L137
But because of in the code the type of `$this->context` is `\Interop\Amqp\AmqpContext` the IDE shows that there is no such method.
So, everything works correctly.